components/elements/Button/Button.js

```diff
@@ -0,0 +1,41 @@
+import styled from 'styled-components';
+import colors from '../../../styles/colors';
+
+const utils = {
+  width: '0.25em',
```

Comment:
width of what?
opacity of what? 
color of what?
Be more specific in variable names